samples/subsys/ipc/openamp/README.rst

```diff
@@ -27,17 +43,31 @@ following settings:
 - Stop bits: 1
 
 Reset the board and the following message will appear on the corresponding
-serial port:
+serial port, one is master another is slave:
```

Comment:
should ``slave`` be ``remote``, since that's what shown in the sample output below?

Reply:
Hi David
Fixed now. Thanks for your review.